lints.rs

```diff
@@ -13,36 +13,45 @@
     clippy::exit,
     clippy::explicit_into_iter_loop,
     clippy::filter_map_next,
+    clippy::float_cmp_const,
```

Comment:
have tested `float_cmp_const` before, but didn't catch anything in our codebase, and when we don't find a lint catching anything we typically don't enable it. at least not right away.

but do think it can be good lint in general, and would be interesting to test on the rest of our code and see, likely can be added

Reply:
I don't think we had any triggering this just yet but we want to prevent it from showing up in the future. Chances that a single explicit floating point constant is expected are low.